pre-cache/check_space

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+APISERVER=https://kubernetes.default.svc
+SERVICEACCOUNT=/var/run/secrets/kubernetes.io/serviceaccount
+NAMESPACE=$(cat ${SERVICEACCOUNT}/namespace)
+TOKEN=$(cat ${SERVICEACCOUNT}/token)
+CACERT=${SERVICEACCOUNT}/ca.crt
+curl --cacert ${CACERT} --header "Authorization: Bearer ${TOKEN}" -X GET ${APISERVER}/api/v1/nodes/${NODE_NAME}/proxy/configz > /tmp/kubeletconfig
```

Comment:
@vitus133 Does the node name really have to be passed in? I imagine it must be on the node itself somewhere?

Reply:
I think only the hostname is on the node. Not necessarily equals to the node name. Using downward API for passing the node name is one  of the examples given in the k8s documentation, so I assume it is also the recommended way
https://kubernetes.io/docs/concepts/workloads/pods/downward-api/